external/loaders/loaders/batches/_batch.py

```diff
@@ -52,6 +52,7 @@ def batches_from_mapper(
     drop_nans: bool = False,
     shuffle_timesteps: bool = True,
     shuffle_samples: bool = False,
+    data_transforms: Optional[Sequence[Mapping]] = None,
```

Comment:
At this point, the `batches_from_mapper` func is just getting passed the raw dict, so it would not be accurate to type-hint with the DataTransform object. If/when we define a `BatchesFromMapperConfig` we could improve this.